common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import get from 'lodash/get';
 import { Form, Input, Button } from 'antd/lib';
+import { Select } from 'antd';
```

Comment:
Just noticed: this also needs to be imported from `antd/lib` or we'll lose the styling from the `autonolas-frontend-library`. Can just move the Select into the import above.